android/build.gradle

```diff
@@ -18,12 +18,13 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
+    namespace 'com.cotecna.capacitor.mediastore'
     compileSdkVersion project.hasProperty('compileSdkVersion') ? rootProject.ext.compileSdkVersion : 32
     defaultConfig {
-        minSdkVersion project.hasProperty('minSdkVersion') ? rootProject.ext.minSdkVersion : 22
+        minSdkVersion project.hasProperty('minSdkVersion') ? rootProject.ext.minSdkVersion : 29
```

Comment:
The Default minSdkVersion in all Capacitor 6 libraries must be 22 not 29.
![image](https://github.com/Cotecna-Inspection/capacitor-mediastore/assets/41290567/4ff8c8ea-7895-4317-85ae-744839518c33)